tests/src/test_tram.cpp

```diff
@@ -0,0 +1,169 @@
+#include <random>
+#include <pybind11/embed.h>
+#include <catch2/catch.hpp>
+#include <deeptime/markov/tram/tram.h>
+//
+// Created by Maaike on 01/12/2021.
+//
+using namespace deeptime;
+
+auto generateDtraj(int nMarkovStates, int length, float shift) {
+    std::vector<float> weights(nMarkovStates, 0.);
```

Comment:
this could be a good example dataset in the `deeptime.data` module don't you think?

Reply:
Is that a place to put datasets that we would use for nothing else but this test? The data doesn't make much physical sense and is only used to unit test, not to check if we actually output correct results.
If we want a tram dataset I would sooner make a TRAM-compatible double well set (with bias matrix) that I would use in the jupyter notebook

Reply:
That sounds better!